docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,471 @@
+```bash
+# Download Anaconda installation script 
+wget -P /tmp https://repo.anaconda.com/archive/Anaconda3-2020.02-Linux-x86_64.sh
+
+# Execute the script to install conda
+bash /tmp/Anaconda3-2020.02-Linux-x86_64.sh
+
+# Please type this command in your terminal to activate Conda environment
+source ~/.bashrc
+``` 
+
+### 2.2.2 Use Conda to Install BigDL and Other Python Libraries
+Create a conda environment, install BigDL and all the needed Python libraries in the created conda environment:
+``` bash
+# "env" is conda environment name, you can use any name you like.
+# Please change Python version to 3.6 if you need a Python 3.6 environment.
+conda create -n env python=3.7 
+conda activate env
+```
+Alternatively, You can install the latest release version of BigDL (built on top of Spark 2.4.6 by default) as follows:
+```bash
+pip install bigdl
+```
+You can install the latest nightly build of BigDL as follows:
+```bash
+pip install --pre --upgrade bigdl
+```
+
+__Note:__ 
+* Using Conda to install BigDL will automatically install libraries including `conda-pack`, `torch`, `torchmetrics`, `torchvision`, `pandas`, `numpy`, `pyspark==2.4.6`, and etc.
+* You can install BigDL built on top of Spark 3.1.2 as follows:
+    
+    ```bash
+    # Install the latest release version
+    pip install bigdl-spark3
+
+    # Install the latest nightly build version
+    pip install --pre --upgrade bigdl-spark3 
+    ```
+    Installing bigdl-spark3 will automatically install `pyspark==3.1.2`.
+* You also need to install any additional python libraries that your application depends on in this Conda environment.
+
+Please see more details in [Python User Guide](https://bigdl.readthedocs.io/en/latest/doc/UserGuide/python.html).
+
+## 2.3 Notes for CDH Users
+* For CDH user, the value of environment variable `HADOOP_CONF_DIR` should be `/etc/hadoop/conf` by default.
+
+* The __Client Node__ (where you submit applications) may have already installed a different version of Spark then the one installed with BigDL. To avoid conflicts, unset all Spark-related environment variables (you may use use `env | grep SPARK` to find all of them):
+    ```bash
+    unset SPARK_HOME
+    unset SPARK_VERSION
+    unset ...
+    ```
+
+* You also need to properly export the location of `libhdfs.so` if needed. For instance, when using pyarrow, set up the environment variable `ARROW_LIBHDFS_DIR` on the __Client Node__ (where you submit the program to the YARN cluster) as follows:
+    ```bash
+    export ARROW_LIBHDFS_DIR=/opt/cloudera/parcels/CDH-${CHD_VERSION}/lib64/
+    ```
+
+# 3. Prepare Dataset 
+To run the example on YARN, you should upload the Fashion-MNIST dataset to a distributed storage (such as HDFS or S3).   
+
+First, please download the Fashion-MNIST dataset manually on your __Client Node__ (where you submit the program to YARN).
```

Comment:
it doesn't have to be the client node

Reply:
agree. Maybe change to `for example, you can first download it on the client node and upload to the distributed storage`